pkg/alertmanager/alertmanager.go

```diff
@@ -370,6 +373,10 @@ func (am *Alertmanager) mergePartialExternalState(part *clusterpb.Part) error {
 	return am.state.(*state).MergePartialState(part)
 }
 
+func (am *Alertmanager) getFullState() (*clusterpb.FullState, error) {
+	return am.state.(*state).GetFullState()
```

Comment:
Check if `am.state` is actually a `*state`. If not, return `error`.

Reply:
:+1: Fix for the line above https://github.com/cortexproject/cortex/pull/3970